src/test/java/uk/ac/ebi/subs/validator/coordinator/SampleValidationMessageEnvelopeExpanderTest.java

```diff
@@ -45,6 +59,9 @@
     @Autowired
     SampleValidationMessageEnvelopeExpander sampleValidatorMessageEnvelopeExpander;
 
+    @MockBean
+    private RestTemplate restTemplate;
```

Comment:
I dont think mocking this object here does anything. In the constructor of the class SimpleValidationMessageEnvelopeExpander, the template object is instantiated directly. This bean here would be useful only if the RestTemplate bean was being injected into the class.

Reply:
Agreed. Fixed and with another refactor moved it to another class, anyway